app/models/manageiq/providers/redfish/inventory/persister/definitions/physical_infra_collections.rb

```diff
@@ -13,5 +13,41 @@ def initialize_physical_infra_collections
     ).each do |name|
       add_collection(physical_infra, name)
     end
+
+    add_physical_server_nics
+    add_physical_server_storages
+    add_physical_server_firmwares
+  end
+
+  private
+
+  def add_physical_server_nics
+    add_collection(physical_infra, :nics) do |builder|
+      builder.add_properties(
+        :model_class                  => ::GuestDevice,
+        :manager_ref                  => %i[device_type uid_ems],
+        :parent_inventory_collections => %i[physical_servers]
+      )
+    end
+  end
+
+  def add_physical_server_storages
+    add_collection(physical_infra, :storage_adapters) do |builder|
+      builder.add_properties(
+        :model_class                  => ::GuestDevice,
+        :manager_ref                  => %i[device_type uid_ems],
+        :parent_inventory_collections => %i[physical_servers]
+      )
+    end
+  end
+
+  def add_physical_server_firmwares
+    add_collection(physical_infra, :firmwares) do |builder|
+      builder.add_properties(
+        :model_class                  => ::Firmware,
+        :manager_ref                  => %i[name resource],
+        :parent_inventory_collections => %i[physical_servers]
+      )
+    end
```

Comment:
@tadeboro can you add all of these to the core set of inventory collections?  I'm trying to get away from all provider overrides in the persister collection definitions.  (here https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/inventory/persister/builder/physical_infra_manager.rb)